services/user_service.go

```diff
@@ -148,4 +148,57 @@ func (u *UserService) token() (*jwt.Jwt, error) {
 
 	u.jwt = jwt.NewJwt(&config.Auth.Jwt, u.cache)
 	return u.jwt, nil
-}
\ No newline at end of file
+}
+
+func (u *UserService) UpdateUser(ctx context.Context, data *models.UpdateUser, user *datastore.User) (*datastore.User, error) {
+	if err := util.Validate(data); err != nil {
+		return nil, NewServiceError(http.StatusBadRequest, err)
+	}
+
+	user.FirstName = data.FirstName
+	user.LastName = data.LastName
+	user.Email = data.Email
+
+	err := u.userRepo.UpdateUser(ctx, user)
```

Comment:
@Dotunj If I mistakenly pass in the `Password` here, will it update the password?

Reply:
Pass in `Password` from the request payload? No, the password won't be updated.